### DISCUSSION TITLE
List View is not included while in Kivy file

### DISCUSSION THREAD

**Discussion by JohnCTXP** [microsoft/vscode-discussions#1987] [Extension Development QnA]
I installed Python and Kivy into Visual Studio Code, but when I typed the List View property, it won't even show a wrench icon stating that it would be there.

![Screenshot (49)](https://github.com/user-attachments/assets/cb551edd-9f97-48cf-9e85-dc0686ca4b6f)

What should I do?

**Comment by gjsjohnmurray** [comment]
Assuming you are not developing an extension, please read this pinned thread for pointers.

**Reply by JohnCTXP** [reply]
How would I develop an extension for List View widget, so that it can be detected?
